apps/sr-frontend/src/app/Header.tsx

```diff
@@ -42,17 +50,34 @@ const lightCones: ITab = {
   textSuffix: <LightConeChip key="lightConeAdd" />,
 }
 
-function LightConeChip() {
+function RelicChip() {
   const { database } = useDatabaseContext()
-  const [dirty, setDirty] = useForceUpdate()
+  const [tally, setTally] = useState(database.relics.keys.length)
   useEffect(
-    () => database.lightCones.followAny(() => setDirty()),
-    [database, setDirty]
+    () =>
+      database.relics.followAny(
+        (_k, r) =>
+          ['new', 'remove'].includes(r) && setTally(database.relics.keys.length)
+      ),
+    [database]
   )
-  const total = useMemo(
-    () => dirty && database.lightCones.keys.length,
-    [database, dirty]
+  const total = useMemo(() => tally, [tally])
```

Comment:
There is no reason to memoize this, since `tally` is already a state. `useMemo` should be used to "cache" expensive calculations.